Add Teleport Cloud downloads page (#11446)
gravitational/teleport: docs/pages/cloud/downloads.mdx

---
title: Teleport Cloud Downloads
description: Cloud Downloads
---

> The Teleport Cloud currently running Teleport =cloud.version=. We recommend

Reviewer: ```suggestion
Teleport Cloud is currently running Teleport (=cloud.version=). We recommend
```

> running at Teleport =cloud.version= agents or one major version below.

Reviewer: ```suggestion
running Teleport agents at Teleport (=cloud.version=) or one major version below.
```

## Teleport archives

> Teleport archives contain all binaries you will need manually install and run

Reviewer: ```suggestion
The following archives contain all of the binaries you will need manually install and run
```

Teleport.

Description                        | | | | |
-----------------------------------|-|-|-|-|
Linux (Debian/Ubuntu compatible)   | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd64-bin.tar.gz) | [arm64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-arm64-bin.tar.gz) | [armv7](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd-bin.tar.gz) | [i386](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-i386-bin.tar.gz) |
Linux (RHEL/CentOS 7.x compatible) | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd64-centos7-bin.tar.gz) |
macOS                              | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-darwin-amd64-bin.tar.gz) |

## Teleport packages

> Teleport packages allow you to use your operating systems package manager to

Reviewer: ```suggestion
The following packages allow you to use your operating system's package manager to
```

install and run Teleport.

Description                        | | | | |
-----------------------------------|-|-|-|-|

> Linux Debian Packages              | [amd64](https://get.gravitational.com/teleport-ent_(=cloud.version)_amd64.deb) | [arm64](https://get.gravitational.com/teleport-ent_(=cloud.version)_arm64.deb) | [armv7](https://get.gravitational.com/teleport-ent_(=cloud.version)_arm.deb) | [i386](https://get.gravitational.com/teleport-ent_(=cloud.version)_i386.deb) |

Reviewer: ```suggestion
Linux Debian Packages              | [amd64](https://get.gravitational.com/teleport-ent_(=cloud.version=)_amd64.deb) | [arm64](https://get.gravitational.com/teleport-ent_(=cloud.version=)_arm64.deb) | [armv7](https://get.gravitational.com/teleport-ent_(=cloud.version=)_arm.deb) | [i386](https://get.gravitational.com/teleport-ent_(=cloud.version=)_i386.deb) |
```

Linux RPM Packages                 | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.x86_64.rpm) | [arm64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm64.rpm) | [armv7](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm.rpm) | [i386](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.i386.rpm) |
macOS Packages                     | [amd64](https://get.gravitational.com/teleport-(=cloud.version=).pkg) |

## Teleport client-only packages

> Teleport client-only packages contain only binaries that need to be deployed to

Reviewer: ```suggestion
Client-only packages contain only binaries that need to be deployed to
```

> users workstations, like `tsh`.

Reviewer: ```suggestion
users' workstations, like `tsh`.
```

---
title: Teleport Cloud Downloads
description: Cloud Downloads
---

Teleport Cloud is currently running Teleport (=cloud.version=). We recommend
running Teleport agents at Teleport (=cloud.version=) or one major version
below.

## Teleport archives

The following archives contain all of the binaries you will need manually
install and run Teleport.

Binaries include:

- `teleport`: the Teleport binary that you can run within your private networks
  to enable access to resources in your infrastructure
- `tctl`: a client tool for managing Teleport resources
- `tsh`: a client tool for accessing resources within your cluster
- `tbot`: a client tool you will use to associate a bot user with a machine

>   identity.

Reviewer: ```suggestion
  identity
```

Linux (Debian/Ubuntu compatible)   | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd64-bin.tar.gz) | [arm64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-arm64-bin.tar.gz) | [armv7](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd-bin.tar.gz) | [i386](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-i386-bin.tar.gz) |
Linux (RHEL/CentOS 7.x compatible) | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd64-centos7-bin.tar.gz) |
macOS                              | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-darwin-amd64-bin.tar.gz) |

## Teleport packages

The following packages allow you to use your operating system's package manager
to install and run Teleport.

Binaries include:

- `teleport`: the Teleport binary that you can run within your private networks
  to enable access to resources in your infrastructure
- `tctl`: a client tool for managing Teleport resources
- `tsh`: a client tool for accessing resources within your cluster
- `tbot`: a client tool you will use to associate a bot user with a machine

>   identity.

Reviewer: ```suggestion
  identity
```
